ibc_data/contrasts/ColumbiaCards.csv

```diff
@@ -0,0 +1,4 @@
+condition
+num_loss_cards

```

Comment:
Not sure how to express this *probability of losing in gambling* and the rest.

Reply:
In that case the contrasts are the conditions.

Reply:
I modified it, I hope this is the correct way!